numpy/core/numeric.py

```diff
@@ -720,6 +721,10 @@ def correlate(a, v, mode='valid'):
 
     which is related to ``c_{av}[k]`` by ``c'_{av}[k] = c_{av}[-k]``.
 
+    The convolution can often be calcualted much faster using ``scipy.signal.correlate``.
+    ``numpy.correlate`` performs slowly because it does not use FFT; for large arrays (i.e. n = 1e5) ``scipy.signal.correlate`` might be considered.
```

Comment:
```suggestion
    `numpy.correlate` may perform slowly in large arrays (i.e. n = 1e5) because it does not use the FFT to compute the convolution; in that case, `scipy.signal.correlate` might be considered.
```

I'm suggesting a rewording to eliminate redundancy, but feel free to reorganize if you think it's better. Also, using single backticks actually generates links to the referenced documentation.